Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM rails:4.2.1
+VOLUME ["/usr/src/app"]
+WORKDIR /usr/src/app
+ADD Gemfile /usr/src/app/Gemfile
```

Comment:
The `Gemfile.lock` is auto-generated by Bundler whenever a gem version changes that means that not including it would do make the build take longer as it would pull the latest version of the gem each and every gem each time we bring up the container.